src/helix-ui/elements/HXFileTileElement.less

```diff
@@ -1,43 +1,154 @@
+@import (reference) "mixins";
 @import "./HXElement";
-@import "elements/hx-icon";
 @import "base/links";
+@import "elements/hx-icon";
+@import "elements/hx-progress";
+@import "components/progress";
+@import (reference) "components/buttons";
 
-#hxFileTile {
-  display: flex;
-  height: 100%;
-  width: 100%;
+// TODO: double check in IE and Edge
 
-  > * {
-    padding: 0.5rem;
+:host {
```

Comment:
Double-check this syntax for polyfilled browsers. I can't remember if this was an issue or not.